scripts/release/snapshot.diff

```diff
@@ -89,6 +103,46 @@
 +        cli_example = display::cli(r#"cargo marker --lints "marker_lints = '0.1.0'""#),
          lints = "--lints".blue(),
 
+=== docs/book/src/usage/ci.md ===
+ ```yml
+-- uses: rust-marker/marker@v<version-major-minor>
++- uses: rust-marker/marker@v0.1
+ ```
+ 
+-- Sliding `v<version-major-minor>` *(recommended)*. Use this to get automatic patch updates.
+-- Fixed `v<version>`. Use this if you want to pin a specific version if there is a regression in a new patch version. If there is one, please create a [new issue], patch versions must never break anything!
++- Sliding `v0.1` *(recommended)*. Use this to get automatic patch updates.
++- Fixed `v0.1.0`. Use this if you want to pin a specific version if there is a regression in a new patch version. If there is one, please create a [new issue], patch versions must never break anything!
+ 
+     --show-error \
+-    --retry <version-major> \
++    --retry 0 \
```

Comment:
This could also become problematic. I wonder if it would be better to just ignore major versions and only do `major.minor` ones. :thinking: 

Or if we want to update the major one automatically, we could expect the `v` prefix? :thinking:

Reply:
Yeah, I've restricted major versions to `v{num}` syntax for now